### PR TITLE
ROS2 port

### LINKED ISSUE #4: Port to ROS2

### PR DESCRIPTION
**Pull request by nkhedekar**
Fixes #4

### CODE REVIEW

File: open3d_conversions/package.xml [hunk 1]
```diff
@@ -12,15 +13,12 @@
 
   <author email="matnay17@gmail.com">Pranay Mathur</author>
   <author email="nkhedekar@nevada.unr.edu">Nikhil Khedekar</author>
+  <buildtool_depend>ament_cmake_ros</buildtool_depend>
 
-  <buildtool_depend>catkin</buildtool_depend>
-  <depend>roscpp</depend>
+  <depend>rclcpp</depend>
   <depend>sensor_msgs</depend>
-  <depend>eigen</depend>
```
Comment (SteveMacenski) [inline open3d_conversions/package.xml:19]: You still use Eigen3, needs to have a depend

**Review by SteveMacenski — COMMENTED** [review]
LGTM. I didn't look at the code changes in the file since I assumed you know what you're doing ;-) 

Other things you should consider is enabling the automatic linters and linting the code. I see a bunch of uncrustify and cpplint violations.

File: open3d_conversions/package.xml (hunk 1 above)
Comment (nkhedekar) [reply open3d_conversions/package.xml:19]: Fixed in d09626d

**Comment by nkhedekar** [comment]
Uncrustify seems to be fine: 

```bash
$ ament_uncrustify .
No code style divergence in file 'perception_open3d/open3d_conversions/include/open3d_conversions/open3d_conversions.hpp'

No code style divergence in file 'perception_open3d/open3d_conversions/include/open3d_conversions/visibility_control.h'

No code style divergence in file 'perception_open3d/open3d_conversions/src/open3d_conversions.cpp'

No code style divergence in file 'perception_open3d/open3d_conversions/test/test_open3d_conversions.cpp'

No problems found

```

**Review by SteveMacenski — APPROVED** [review]
Try running with all the linters, the autolint ament macro will run this. I see a white space (at minimum in package.xml)

**Comment by nkhedekar** [comment]
- Removed extra spaces
- Removed unused variables (using `-Wall -Wextra -Wpedantic`)
- Checked with `ament_cpplint`:
```bash
$ ament_cpplint .
Using '--root=/home/nkhedekar/Desktop/workspaces/o3d_ws/src/perception_open3d/open3d_conversions/include' argument

Done processing /home/nkhedekar/Desktop/workspaces/o3d_ws/src/perception_open3d/open3d_conversions/include/open3d_conversions/open3d_conversions.hpp

Done processing /home/nkhedekar/Desktop/workspaces/o3d_ws/src/perception_open3d/open3d_conversions/include/open3d_conversions/visibility_control.h

Using '--root=/home/nkhedekar/Desktop/workspaces/o3d_ws/src/perception_open3d/open3d_conversions/src' argument

Done processing /home/nkhedekar/Desktop/workspaces/o3d_ws/src/perception_open3d/open3d_conversions/src/open3d_conversions.cpp

Using '--root=/home/nkhedekar/Desktop/workspaces/o3d_ws/src/perception_open3d/open3d_conversions/test' argument

Done processing /home/nkhedekar/Desktop/workspaces/o3d_ws/src/perception_open3d/open3d_conversions/test/test_open3d_conversions.cpp

No problems found
```
- Checked with `ament_cppcheck`

```bash
$ ament_cppcheck .
No problems found
```

- Checked with `ament_uncrustify`

```bash
$ ament_uncrustify .
No code style divergence in file 'include/open3d_conversions/open3d_conversions.hpp'

No code style divergence in file 'include/open3d_conversions/visibility_control.h'

No code style divergence in file 'src/open3d_conversions.cpp'

No code style divergence in file 'test/test_open3d_conversions.cpp'

No problems found
```

**Pull request merged**